libraries/stdlib/jvm/src/kotlin/text/StringsJVM.kt

```diff
@@ -630,35 +664,60 @@ public inline fun String.toPattern(flags: Int = 0): java.util.regex.Pattern {
  *
  * The title case of a character is usually the same as its upper case with several exceptions.
  * The particular list of characters with the special title case form depends on the underlying platform.
- *
- * @sample samples.text.Strings.capitalize
  */
 public actual fun String.capitalize(): String {
     return capitalize(Locale.getDefault())
 }
 
+/**
+ * Returns a copy of this string having its first letter titlecased using Unicode mapping rules of the invariant locale,
+ * or the original string if it's empty or already starts with a title case letter.
+ *
+ * @sample samples.text.Strings.capitalizeFirst
+ */
+@SinceKotlin("1.4")
+@ExperimentalStdlibApi
+public actual fun String.capitalizeFirst(): String {
+    return capitalizeFirst(Locale.ROOT)
+}
+
 /**
  * Returns a copy of this string having its first letter titlecased using the rules of the specified [locale],
  * or the original string if it's empty or already starts with a title case letter.
  *
  * The title case of a character is usually the same as its upper case with several exceptions.
  * The particular list of characters with the special title case form depends on the underlying platform.
  */
+@OptIn(ExperimentalStdlibApi::class)
 @SinceKotlin("1.4")
 @WasExperimental(ExperimentalStdlibApi::class)
 @kotlin.internal.LowPriorityInOverloadResolution
```

Comment:
Not sure why this `@ kotlin.internal.LowPriorityInOverloadResolution` was needed

Reply:
It was added to avoid conflicts in function references without the expected type, like `val f = ::capitalize`, that would have otherwise broken after introducing an overload to the existing single function.

Reply:
Might be worth a code comment about that fact then?

Reply:
done